Projects/UOContent/Mobiles/Townfolk/Banker.cs

```diff
@@ -103,6 +109,51 @@ public static int GetBalance(Mobile m, out List<Item> gold, out List<Item> check
         return (int)Math.Clamp(balance, 0, int.MaxValue);
     }
 
+    private static bool HasRequiredBalance(int requiredBalance, Mobile m, out PooledRefList<Gold> gold, out PooledRefList<BankCheck> checks)
```

Comment:
We can probably get away with a `PooledRefQueue` - Which I think is slightly more performant? - I would keep it a list if this function was public. 🤔